app/src/main/res/values/strings.xml

```diff
@@ -18,330 +18,334 @@
+    <string name="education_select_database_summary">Open your earlier database file from your file browser to continue using it.</string>
     <string name="education_open_link_database_title">A link to the location of your file is sufficient</string>
     <string name="education_open_link_database_summary">You can also open your database with a physical link (with file:// and content:// for example).</string>
-    <string name="education_new_node_title">Add new items to your base</string>
-    <string name="education_new_node_summary">Add entries to manage your digital identities.\n\nAdd groups (the equivalent of folders) to organize your entries and your database.</string>
-    <string name="education_search_title">Easily search your entries</string>
-    <string name="education_search_summary">Search for entries by title, username or other fields to easily retrieve your passwords.</string>
-    <string name="education_fingerprint_title">Unlock your database with your fingerprint</string>
-    <string name="education_fingerprint_summary">Make the link between your password and your fingerprint to easily unlock your database.</string>
-    <string name="education_entry_edit_title">Edit the entry</string>
-    <string name="education_entry_edit_summary">Edit your entry with custom fields, references to pool data can be added between fields of different entries.</string>
-    <string name="education_generate_password_title">Create a strong password</string>
-    <string name="education_generate_password_summary">Generate a strong password to associate with your entry, easily define it according to the criteria of the form and don\'t forget secure password you can remember.</string>
+    <!--both entries and groups are items-->
+    <string name="education_new_node_title">Add items to your database</string>
+    <string name="education_new_node_summary">Entries help manage your digital identities.\n\nGroups (~ folders) organize entries in your database.</string>
+    <string name="education_search_title">Search through entries</string>
+    <string name="education_search_summary">Enter title, username or content of other fields to retrieve your passwords.</string>
+    <string name="education_fingerprint_title">Database Unlocking by Fingerprint</string>
+    <string name="education_fingerprint_summary">Link your password to your scanned fingerprint to quickly unlock your database.</string>
+    <string name="education_entry_edit_title">Edit the Entry</string>
+    <string name="education_entry_edit_summary">Edit your entry with custom fields. Pool data can be referenced between different entry fields.</string>
+    <string name="education_generate_password_title">Create a strong password for your entry.</string>
+    <string name="education_generate_password_summary">Don\'t forget to add a title you remember your entry by.</string>
```

Comment:
Sentence not finished?

Reply:
What one remembers the title by, is the memorable title.

Reply:
OK my bad!